cvat/apps/engine/migrations/0020_remove_task_flipped.py

```diff
@@ -0,0 +1,125 @@
+# Generated by Django 2.1.7 on 2019-06-18 11:08
+
+from django.db import migrations
+from django.conf import settings
+
+from cvat.apps.engine.task import get_image_meta_cache
+from cvat.apps.engine.models import Job, ShapeType
+
+from PIL import Image
+
+import os
+
+
+def _flip_shape(shape, size):
+    if shape.type == ShapeType.RECTANGLE:
+        shape.points = [
+            shape.points[2], # xbr -> xtl
+            shape.points[3], # ybr -> ytl
+            shape.points[0], # xtl -> xbr
+            shape.points[1]  # ytl -> ybr
+        ]
+
+    for x in range(0, len(shape.points), 2):
+        y = x + 1
+        shape.points[x] = size['width'] - shape.points[x]
+        shape.points[y] = size['height'] - shape.points[y]
+
+
+def frame_path(db_task, frame):
```

Comment:
@bsekachev , can we avoid the code duplication?

Reply:
@nmanovic 
In theory we can pop these methods from existing code. But there is a reason do not do it. 
The database is always predefined for a migration, i.e. a migration knows about it's structure. We can't say the same about external code. After 10 commits it can be changed while a migration code will be waiting for old version of code. So migration won't work.